<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DoubleCross</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_main">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="TextWidget" name="text">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget_right" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_right">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="widget_analytic" native="true">
           <layout class="QGridLayout" name="gridLayout" columnminimumwidth="1,4,4">
            <item row="0" column="0" rowspan="2">
             <widget class="StatusWidget" name="status" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7"/>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="character">
              <property name="text">
               <string>Character : 0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="word">
              <property name="text">
               <string>Word : 0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="sentence">
              <property name="text">
               <string>Sentence : 0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_word" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="WordWidget" name="table_word"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QRadioButton" name="synonym">
                 <property name="text">
                  <string>Synonyms</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="antonym">
                 <property name="text">
                  <string>Antonyms</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_search">
                 <property name="text">
                  <string>Search</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="search"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="ExplainWidget" name="explain">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_explain">
            <attribute name="title">
             <string>About</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QTextBrowser" name="tab_initial">
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;English Redundancy Checker&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Made by Jeuk Hwang&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;License : &lt;a href=&quot;https://opensource.org/licenses/MIT&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;MIT&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Contact : &lt;a href=&quot;https://github.com/PositivePeriod/DoubleCross&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/PositivePeriod/DoubleCross&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Word resource : &lt;a href=&quot;https://www.thesaurus.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://www.thesaurus.com/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Rule : character = the length of text / word = divided by space / sentence = the number of dot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHowtouse"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCheck">
    <property name="title">
     <string>Check</string>
    </property>
    <addaction name="actionCheck"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionArialFont"/>
     <addaction name="actionCalibriFont"/>
     <addaction name="actionGulimFont"/>
     <addaction name="actionTimesFont"/>
     <addaction name="separator"/>
     <addaction name="actionSelectFont"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_word_data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCheck"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionHowtouse">
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReset_Whitelist">
   <property name="text">
    <string>Reset whitelist</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="actionTimes_New_Roman">
   <property name="text">
    <string>Times New Roman</string>
   </property>
  </action>
  <action name="actionGulim">
   <property name="text">
    <string>Gulim</string>
   </property>
  </action>
  <action name="actionCalibri">
   <property name="text">
    <string>Calibri</string>
   </property>
  </action>
  <action name="actionCalibri_Light">
   <property name="text">
    <string>Calibri Light</string>
   </property>
  </action>
  <action name="actionSelectFont">
   <property name="text">
    <string>More fonts...</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionBright">
   <property name="text">
    <string>Bright</string>
   </property>
  </action>
  <action name="actionStopword">
   <property name="text">
    <string>Exception</string>
   </property>
  </action>
  <action name="actionException">
   <property name="text">
    <string>Exception</string>
   </property>
  </action>
  <action name="actionWill_be_added">
   <property name="text">
    <string>Will be added</string>
   </property>
  </action>
  <action name="actionArialFont">
   <property name="text">
    <string>Arial</string>
   </property>
  </action>
  <action name="actionCalibriFont">
   <property name="text">
    <string>Calibri</string>
   </property>
  </action>
  <action name="actionGulimFont">
   <property name="text">
    <string>Gulim</string>
   </property>
  </action>
  <action name="actionTimesFont">
   <property name="text">
    <string>Times New Roman</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionErase_searched_data">
   <property name="text">
    <string>Erase searched data</string>
   </property>
  </action>
  <action name="actionDelete_search_data">
   <property name="text">
    <string>Delete search data</string>
   </property>
  </action>
  <action name="actionDelete_word_data">
   <property name="text">
    <string>Delete word data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusWidget</class>
   <extends>QWidget</extends>
   <header>./script/custom_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExplainWidget</class>
   <extends>QTabWidget</extends>
   <header>./script/custom_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WordWidget</class>
   <extends>QTableWidget</extends>
   <header>./script/custom_widget.h</header>
  </customwidget>
  <customwidget>
   <class>TextWidget</class>
   <extends>QTextEdit</extends>
   <header>./script/custom_widget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>text</tabstop>
  <tabstop>explain</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
